feeadjuster/feeadjuster.py

```diff
@@ -96,9 +96,12 @@ def get_fees_median(plugin: Plugin, scid: str):
     """
     peer_id = get_peer_id_for_scid(plugin, scid)
     assert peer_id is not None
+    if plugin.listchannels_by_dst:
+        plugin.channels = plugin.rpc.call("listchannels",
+                                          {"destination": peer_id})['channels']
```

Comment:
```suggestion
    if plugin.channels == None:
        if plugin.listchannels_by_dst:
            plugin.channels = plugin.rpc.call("listchannels",
                                              {"destination": peer_id})['channels']
        else:
            plugin.channels = plugin.rpc.listchannels()['channels']
```

Reply:
Well, incase we use listchannels_by_dst we need to set it to None for the next channel after calculating the median fees, otherwise it will use an incorrect list (not for that next channel).

Sidenote: I already figured for ~30 channels on an raspberry pi this takes about 15secs for the initial loop when calling `listchannels(destinaion)` on a per channel basis. But thats not much difference compared to saving all channels once and filtering in python for each channel.